what-the-stack/src/main/java/com/haroldadmin/whatthestack/WhatTheStackActivity.kt

```diff
@@ -7,11 +7,9 @@ import android.content.Intent
 import android.os.Bundle
 import android.text.method.ScrollingMovementMethod
 import android.view.View
-import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
-import androidx.appcompat.widget.AppCompatTextView
-import com.google.android.material.button.MaterialButton
 import com.google.android.material.snackbar.Snackbar
+import com.haroldadmin.whatthestack.databinding.ActivityWhatTheStackBinding
 import dev.chrisbanes.insetter.Insetter
 import dev.chrisbanes.insetter.Side
 import kotlinx.android.synthetic.main.activity_what_the_stack.*
```

Comment:
Remove the synthetics import, since its not required.

Reply:
done